lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -239,6 +239,31 @@ static size_t setKernelArgsForBuffers(cl_kernel kernel, const Instruction &I,
   return kernelArgIdx - 1;
 }
 
+unsigned getPreferredVectorWidth(cl_device_id device, ElemKind elemTy) {
```

Comment:
Could you make is static and add a Doxygen comment?

Reply:
Done.